poly-synth/index.html

```diff
@@ -20,23 +20,55 @@
     <script src="src/poly-synth-demo.js"></script>
     <script src="src/poly-synth.js"></script>
     <script src="src/poly-synth-voice.js"></script>
+    <style>
+      #container {
+        display: flex;
+        flex-direction: row;
+      }
+      .controls {
+        border-style: solid;
+        margin: 5px;
+      }
+    </style>
   </head>
   <body>
+    <h1>Polyphonic Subtractive Synthesizer</h1>
```

Comment:
I couldn't find a sublime text package called HTML tidy but I noticed that [SublimeLinter](http://www.sublimelinter.com/en/latest/) had an HTML tidy component. I ran that and it gave warnings about these lines 
```
<script src="third-party/qwerty-hancock.min.js"></script>
    <script src="../audio-worklet/lib/param-controller.js"></script>
    <script src="src/poly-synth-demo.js"></script>
    <script src="src/poly-synth.js"></script>
    <script src="src/poly-synth-voice.js"></script>
``` saying inserting "type" attribute. It wasn't complaing about the line you reference. What is wrong with it? And am I using the correct html-tidy program?